spec/requests/admin_settings_request_spec.rb

```diff
@@ -0,0 +1,25 @@
+require "spec_helper"
+
+describe "Admin settings", type: :request do
```

Comment:
You can omit `type: :request`, since the test is in `spec/requests` and we use `infer_spec_type_from_file_location!`.